src/filing/submission/Receipt.jsx

```diff
@@ -5,47 +5,58 @@ import PrintPageButton from './PrintPageButton'
 import { isBeta } from '../../common/Beta.jsx'
 import { ordinalHour } from '../utils/date.js'
 import { SIGNED } from '../constants/statusCodes.js'
+import { parseTimedGuardDate } from '../../deriveConfig.js'
+
+const Receipt = ({ status, timestamp, receipt, filingPeriod, email }) => {
+  const options = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    timezone: 'America/New_York',
+  }
+
+  let daySubmitted = new Date(timestamp).toLocaleDateString('en-US', options)
+
+  let easternTime = new Date(timestamp).toLocaleString('en-US', {
+    timeZone: 'America/New_York',
+  })
+
+  let timeOfReceipt = daySubmitted + ',' + easternTime.split(',')[1] + " ET"
```

Comment:
Let's extract this into a `formatReceiptTime` function. Then on line 54 we can just 
```
<strong>{formatReceiptTime(timestamp)}</strong>
```
 
 I would parse the date once, then reuse: `const date = new Date(timestamp)`

Reply:
Logic has been extracted from `Receipt` component and now resides in `utils/date` as a function.